examples/host-vars.yml

```diff
@@ -26,3 +26,15 @@ matrix_coturn_turn_static_auth_secret: ""
 # You can put any string here, but generating a strong one is preferred (e.g. `pwgen -s 64 1`).
 matrix_synapse_macaroon_secret_key: ""
 
+# Set autojoin room(s)
```

Comment:
All default variables are meant to be specified in `roles/matrix-server/defaults/main.yml`.

We have `examples/host-vars.yml` to serve as a bare-minimum example of what people **must** define in order to get this playbook working. Besides that, people can override any of the many defaults from `roles/matrix-server/defaults/main.yml`.

So all of these ought to be moved to  `roles/matrix-server/defaults/main.yml`.
Not only that, but they'd better have sensible default values, so that the majority of people would not have to change them. Keeping backward compatibility is nice too. Not that we couldn't break it, if it makes sense.

I guess `riot_roomdir_servers` can be defined as `['matrix.org']` to preserve compatibility with what we have now and to provide a nice default.

`matrix_autojoin_rooms` can be defined as an empty list (`[]`) for the same reason.